blockchain/transaction.go

```diff
@@ -88,3 +89,23 @@ func (t *Transaction) Marshal() []byte {
 	buf = append(buf, t.Sig.Marshal()...)
 	return buf
 }
+
+// InputsEqualOutputs returns true if t.Inputs == other.Outputs, as well
+// as the difference between the two (outputs - inputs).
+func (t *Transaction) InputsEqualOutputs(other ...*Transaction) (bool, int) {
```

Comment:
If we aren't using it, I'd leave it out until we need it. Fees will eventually be part of the transaction so we can verify as part of this function anyway right?